src/main/java/com/hackathonorganizer/hackathonwriteservice/team/service/TagService.java

```diff
@@ -0,0 +1,73 @@
+package com.hackathonorganizer.hackathonwriteservice.team.service;
+
+import com.hackathonorganizer.hackathonwriteservice.team.exception.ResourceAlreadyExistsException;
+import com.hackathonorganizer.hackathonwriteservice.team.exception.ResourceNotFoundException;
+import com.hackathonorganizer.hackathonwriteservice.team.model.Tag;
+import com.hackathonorganizer.hackathonwriteservice.team.model.dto.TagRequest;
+import com.hackathonorganizer.hackathonwriteservice.team.repository.TagRepository;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import lombok.val;
+import org.springframework.stereotype.Service;
+
+@Service
+@RequiredArgsConstructor
+@Slf4j
+public class TagService {
+
+    private final TagRepository tagRepository;
+
+    public Tag create(TagRequest tagRequest) {
+        validateName(tagRequest.name());
+        val tagToSave = Tag.builder()
```

Comment:
Ten kod mozna przeniesc do prywatnej metody

Reply:
Buildera? I co to da i jak metoda powinna się nazywać? Dlatego używam buildera żeby było czytelniej, nie widzę sensu wynoszenia samego buildera do metody

Reply:
Zgodnie z clean codem funkcje powinny byc jak najmniejsze